styles/pages/_home.scss

```diff
@@ -96,37 +94,28 @@
 
   .heroboxes {
     margin-top: 200px;
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    align-items: flex-start;
-    clear: both;
-  }
-  .hb1 {
-    margin-right: 35px;
-  }
-  .hb3 {
-    margin-left: 35px;
-  }
-  .herobox {
-    /* Kevin/Roman - These elements and top-borders need to have
-    padding between them, but not padding on the outside edges
-    or else things won't line up with existing container elements. */
-    max-width: 33.3%; /* I know it's garbage */
-    border-top:2px solid #ffffff;
-  }
-  .herobox h1 {
-    font-style: italic;
-    font-family: $font-heading;
-    font-weight:900;
-  }
-  .herobox p {
-    font-family: $font-body;
-    font-size: 16px;
-    line-height:1.45em;
-  }
 
-/* because of the border, these "shift" when hovered? */
+    h1 {
+      font-style: italic;
+      font-family: $font-heading;
+      font-weight:900;
+      font-size: 30px;
+    }
+    p {
+      font-family: $font-body;
+      font-size: 16px;
+      line-height:1.45em;
+    }
+
+    .herobox {
+      border-top:2px solid #ffffff;
+      @include span (4);
+    }
+
+    .hb3 {
```

Comment:
It's either that or we use nth because middle needs to be center-aligned and the 3rd needs to be floated right.

or another thing that I'm not aware of, but since all three aren't equal, I don't believe so.